app-modules/authorization/config/permissions/web/model.php

```diff
@@ -3,6 +3,7 @@
 return [
     'view-any',
     'create',
+    'import',
```

Comment:
@dgoetzit in your opinion does this belong here in the Model permissions, or should it be in `custom`?

Reply:
Done